docker-compose.yml

```diff
@@ -6,6 +6,4 @@ services:
       - -js
     ports:
       - 4222:4222
-networks:
```

Comment:
I was getting a warning from docker-compose that we had defined a network, but no containers were using it, so I dropped this?